etc/parameters/hardware.json

```diff
@@ -96,7 +96,19 @@
   "paths": {
     "motions": "etc/motions",
     "neural_networks": "etc/neural_networks",
-    "parameters": "etc/parameters"
+    "parameters": "etc/parameters",
+    "sounds": "etc/sounds"
+  },
+  "speakers": {
+    "access": "RWInterleaved",
+    "buffer_time": {
+      "nanos": 0,
+      "secs": 1
+    },
+    "format": "FloatLE",
+    "number_of_channels": 1,
```

Comment:
All of our sounds are in Mono and, because of interleaving channels, they would be played back twice as fast if we change the number of channels. Also, why do we need stereo? You can probably not hear from which side the sound came from. Vote for skip.